help.txt

```diff
@@ -32,10 +32,8 @@ PACKAGE CONTENTS
     s3
     smart_open_lib
     ssh
-    tests (package)
     transport
     utils
-    version
```

Comment:
first time https://github.com/piskvorky/smart_open/pull/853 comes in handy: just pulled `develop` and lint started failing on this PR :)